jujugui/static/gui/src/app/components/env-size-display/_env-size-display.scss

```diff
@@ -21,7 +21,7 @@
         flex: 1;
         border: 1px solid $light-mid-grey;
         margin-bottom: 0;
-        background-color: transparent;
+        background-color: $canvas-background;
```

Comment:
Drive by so you can't see the service blocks behind the unselected service/machine toggle buttons.